addons/iot_drivers/webrtc_client.py

```diff
@@ -3,113 +3,117 @@
-                    self.send({
-                        'owner': message['session_id'],
-                        'device_identifier': helpers.get_identifier(),
-                        'time': time.time(),
-                        'status': 'success',
-                    })
-                elif message_type == "restart_odoo":
-                    self.send({
-                        'owner': message['session_id'],
-                        'device_identifier': helpers.get_identifier(),
-                        'time': time.time(),
-                        'status': 'success',
-                    })
-                    await self.event_loop.run_in_executor(None, helpers.odoo_restart)
-
-            @channel.on("close")
-            def on_close():
-                self.connections.discard(channel)
-
-        @peer_connection.on("connectionstatechange")
-        async def on_connectionstatechange():
-            if peer_connection.connectionState == "failed":
-                await peer_connection.close()
-
-        await peer_connection.setRemoteDescription(offer)
-
-        answer = await peer_connection.createAnswer()
-        await peer_connection.setLocalDescription(answer)
-
-        return {"sdp": peer_connection.localDescription.sdp, "type": peer_connection.localDescription.type}
-
-    async def _send(self, data: dict):
-        for connection in self.connections:
-            connection.send(json.dumps(data))
-
-    def run(self):
-        self.event_loop.run_forever()
-
-
-webrtc_client = WebRtcClient()
-webrtc_client.start()
+                    elif message_type == "restart_odoo":
+                        self.send({
+                            'owner': message['session_id'],
+                            'device_identifier': helpers.get_identifier(),
+                            'time': time.time(),
+                            'status': 'success',
+                        })
+                        await self.event_loop.run_in_executor(None, helpers.odoo_restart)
+
+                @channel.on("close")
+                def on_close():
+                    self.connections.discard(channel)
+
+            @peer_connection.on("connectionstatechange")
+            async def on_connectionstatechange():
+                if peer_connection.connectionState == "failed":
+                    await peer_connection.close()
+
+            await peer_connection.setRemoteDescription(offer)
+
+            answer = await peer_connection.createAnswer()
+            await peer_connection.setLocalDescription(answer)
+
+            return {"sdp": peer_connection.localDescription.sdp, "type": peer_connection.localDescription.type}
+
+        async def _send(self, data: dict):
+            for connection in self.connections:
+                connection.send(json.dumps(data))
+
+        def run(self):
+            self.event_loop.run_forever()
+
+    webrtc_client = WebRtcClient()
+    webrtc_client.start()
```

Comment:
The exceptions would only be triggered if the class is instantiated so instead the check can be done only around the class instantiation to avoid the big diff

```Python
if WEBRTC_AVAILABLE:
    webrtc_client = WebRtcClient()
    webrtc_client.start()
```